pkg/chunk/schema_config.go

```diff
@@ -295,6 +295,39 @@ type PeriodicTableConfig struct {
 	Tags   Tags          `yaml:"tags,omitempty"`
 }
 
+// UnmarshalYAML implements the yaml.Unmarshaler interface.
+func (cfg *PeriodicTableConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
```

Comment:
Could you also add a unit test covering the unmarshalling with a non standard duration (ie. `1y`)?